1907-07-01.xml

```diff
@@ -0,0 +1,1213 @@
+                                        <cell/>
+                                    </row>
+                                    <row>
+                                        <cell>Sun. and Wedn.‡</cell>
+                                        <cell>Sat. and Tues.</cell>
+                                        <cell>8 p.m.</cell>
+                                        <cell>depart</cell>
+                                        <cell>Cairo</cell>
+                                        <cell>arrive</cell>
+                                        <cell>Sun. and Wedn.‡ </cell>
+                                        <cell>Tues. and Satur.</cell>
+                                        <cell>7.35 a.m.</cell>
+                                    </row>
+                                    <row>
+                                        <cell>Mon. and Thurs.</cell>
+                                        <cell>Sun. and Wedn.</cell>
+                                        <cell>5.40 p.m..</cell>
+                                        <cell>depart</cell>
+                                        <cell>Shellal</cell>
+                                        <cell>depart</cell>
+                                        <cell>Sat. and Tues.‡</cell>
+                                        <cell>Mon. and Fridays</cell>
+                                        <cell>9.40 a.m.</cell>
+                                    </row>
+                                    <row>
+                                        <cell>Mon. and Thurs.</cell>
+                                        <cell>Sun. and Wedn.</cell>
+                                        <cell>7 p.m..</cell>
+                                        <cell>depart</cell>
+                                        <cell>Shellal</cell>
+                                        <cell>arrive</cell>
+                                        <cell>Sat. and Tues.‡</cell>
+                                        <cell>Mon. and Fridays</cell>
+                                        <cell>mornings</cell>
+                                    </row>
+                                    <row>
+                                        <cell>Wedn. and Sat.</cell>
+                                        <cell>Wedn. and Sat.</cell>
+                                        <cell>11 a.m.</cell>
+                                        <cell>arr.</cell>
+                                        <cell>Halfa</cell>
+                                        <cell>depart</cell>
+                                        <cell>Fri. and Mon. 6 a.m.</cell>
+                                        <cell>Sat and Wedn</cell>
+                                        <cell>noon.</cell>
+                                    </row>
+                                    <row>
+                                        <cell>Wednesdays and Saturdays</cell>
+                                        <cell/>Wednesdays and Saturdays</cell>
```

Comment:
use `<cell>` not `<cell/>`, which closes the cell as soon as it opens it. Similar errors below.